src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
```

Comment:
@mariospr ah, right. I only thought of the file that is touched to indicate Chrome is already installed.
Surely we can add it to GNOME Software but are we sure we always want to make Chrome the default browser when it's installed (it's difficult to detect how the installation was triggered: from the unattended script or the plugin).